bot/persistence/resources/init_db.sql

```diff
@@ -2,3 +2,8 @@ CREATE TABLE IF NOT EXISTS configs(
     config_key varchar PRIMARY KEY,
     val varchar NOT NULL
 );
+
+CREATE TABLE IF NOT EXISTS Modmail(
+    ID INTEGER NOT NULL PRIMARY KEY,
```

Comment:
That's not fully correct. Due to a bug a Primary Key in SQLite is implicitly `Not Null` only if its a `Integer Primary Key` otherwise it's theoratically nullable. 🤡 
https://www.sqlitetutorial.net/sqlite-not-null-constraint/

Discord uses Twitters "Snowflake" format for unique IDs which is generally an unsigned 64Bit Integer. However in the case of Python it doesn't matter how big a value is because in Python, the value of an integer is not restricted by the number of bits and can expand to the limit of the available memory.
https://stackoverflow.com/a/7604981/11889326
**TL;DR: Integers are unbounded in Python.**

Reply:
Oh didn't know that

But are they unbounded in sqllite?

Reply:
No... SQLite can only handle signed 64bit Integers. I'm storing the IDs now as `Text `instead of `Integer`.